test/__tests__/data/pinterest_tag_input.json

```diff
@@ -1412,6 +1412,79 @@
       "Transformations": []
     }
   },
+  {
+    "message": {
```

Comment:
In component tests, 2 test cases - one with `destination.Config.sendingUnHashedData` with true & other with false will suffice

Reply:
In the component tests, it is already there. Should I add unit tests as well?